src/com/mostc/pftt/runner/AbstractLocalTestPackRunner.java

```diff
@@ -898,11 +895,11 @@ protected void exec_jobs(boolean nts, TestCaseGroupKey group_key, LinkedBlocking
 										new_wsi = ((WebServerScenario)sapi_scenario).smgr.getWebServerInstance(cm, runner_fs, runner_host, scenario_set, build, 
 												group_key.getPhpIni(),
 												group_key.getEnv(),
-												this instanceof PhpUnitThread ? //src_test_pack.getSourceDirectory()//
+												// this instanceof PhpUnitThread ? //src_test_pack.getSourceDirectory()//
```

Comment:
If these lines are going to be killed, please lets remove them instead of commenting out.

Thanks.

Reply:
Updated and removed those lines.